README.md

```diff
@@ -104,6 +104,12 @@ Make sure to define `COVERITY_SCAN_TOKEN` in your
     # Default: 'make'
     command: ''
 
+    # Specify the working-directory of where to run commands.
+    # https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsrun
+    #
+    # Default: '${{ github.workspace }}'
+    working_directory: ''
```

Comment:
the standard name in GH actions uses a `-` instead of `_`

Reply:
that's a good point.  GH actions seem to use snake case & kebab case in different situations, but it doesn't seem to document one as preferred over the other.  i think i went with snake case because i was migrating existing code that used it (like the Travis CI syntax), but i don't recall a specific reason for it here other than maybe i assumed kebab-case wasn't valid syntax.

let's go with `working-directory` to match standard GH actions, and at some point i'll do a new release that renames existing `build_xxx` to `build-xxx`.  i'll have to experiment a bit to see if i can handle this in a backwards compatible way, at least for the next release.

Reply:
Rebased this PR, changed to `working-directory` and tested successfully on my side.

Reply:
the failure isn't your fault.  the token used in the self tests is saved in the repo's secrets, but the secrets aren't available to pull requests from external repos, so it fails to download due to auth errors.

i can't figure out how to make the secret available to all pull requests short of hardcoding the token in the yaml file and exposing it everything.  which isn't a big deal since it just goes to this test project, but i also don't want it getting spammed/abused.